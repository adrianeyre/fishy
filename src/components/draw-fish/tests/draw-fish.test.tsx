import React from 'react';
import { shallow } from 'enzyme';

import DrawFish from '../draw-fish';
import IDrawFishProps from '../interfaces/draw-fish-props';
import Fish from '../../../classes/fish';

describe('Draw Fish', () => {
	it('Should render correctly', () => {
		const defaultProps: IDrawFishProps = {
			fish: new Fish({}, 1, 1, 1),
			image: 'image',
		};

		const drawFish = shallow(<DrawFish {...defaultProps} />);
		expect(drawFish).toMatchSnapshot();
	});
});