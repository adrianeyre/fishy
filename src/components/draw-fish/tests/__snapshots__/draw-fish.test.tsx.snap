// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Draw Fish Should render correctly 1`] = `ShallowWrapper {}`;
